bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/WebexAuthService.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.webexteams.internal;
+
+import static org.openhab.binding.webexteams.internal.WebexTeamsBindingConstants.*;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link WebexAuthService} class to manage the servlets and bind authorization servlet to bridges.
+ * 
+ * @author Tom Deckers - Initial contribution
+ */
+@Component(service = WebexAuthService.class, configurationPid = "binding.webex.authService")
+@NonNullByDefault
+public class WebexAuthService {
+
+    private static final String TEMPLATE_PATH = "templates/";
+    private static final String TEMPLATE_ACCOUNT = TEMPLATE_PATH + "account.html";
+    private static final String TEMPLATE_INDEX = TEMPLATE_PATH + "index.html";
+
+    private final Logger logger = LoggerFactory.getLogger(WebexAuthService.class);
+
+    private final List<WebexTeamsHandler> handlers = new ArrayList<>();
+
+    private static final String ERROR_UKNOWN_BRIDGE = "Returned 'state' by doesn't match any accounts. Has the account been removed?";
```

Comment:
Looks like a word is missing after "by" ?

Reply:
Fixed.